packages/clay-css/src/scss/components/_cards.scss

```diff
@@ -418,16 +418,24 @@
 
 .card-page {
 	@extend %clay-custom-grid-wrapper;
-}
 
-.card-page-equal-height {
-	.card-page-item {
-		display: flex;
-		flex-direction: column;
-	}
+	&.card-page-equal-height {
```

Comment:
What use cases should we add this to ClayCardGrid?

Reply:
This is UX preference. I can't think of a rule that would cover every case. In Portal, I would say all the cards should have this class except for http://pat270.github.io/lexicon/v2.10.0/content/test-card-custom-fields-view/